common/logisticspipes/blocks/powertile/LogisticsPowerJunctionTileEntity.java

```diff
@@ -121,8 +127,39 @@ public boolean canReceive() {
 		}
 	};
 
+	private Object mjReceiver;
+
 	public LogisticsPowerJunctionTileEntity() {
 		HUD = new HUDPowerLevel(this);
+
+		if (Loader.isModLoaded("buildcraftlib")) {
+			initMjReceiver();
+		}
+	}
+
+	// Split off into its own method to avoid JVM shenanigans when IMjReceiver is not there.
+	private void initMjReceiver() {
```

Comment:
Would you mind moving this into the buildcraft proxy? 
Pass in this TE and call it something like createMjReceiver which returns the receiver as an Object or null in the dummy proxy.